PNG provide credential examples

PatStLouis opened the pull request.

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/governance.md:
```diff
@@ -131,120 +131,35 @@ Legislation and regulations govern the disposition, administration, and manageme
 
-## 2.6. Technical Requirements (Credential)
+## 2.6 Technical Requirements (Credential)
 
```
```suggestion
Technical requirements include both Json-LD and Anoncred formats to enable interoperability with multiple formats: 

The Json-LD credential format is designed to meet W3C VCDM 2.0 requirements and is modeled for compliance with the [UN Transparency Protocol](https://uncefact.github.io/spec-untp/docs/about/) [Digital Conformity Credential](https://uncefact.github.io/spec-untp/docs/specification/ConformityCredential).

The Anoncreds credential format enables the exchange of credential data between digital wallets once a trusted connection is established. 
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+A stratum or strata designated by the regulator as a zone generally or for a designated area or a specific well, or a stratum or strata designated by the director for the purposes of the administration of petroleum and natural gas rights or storage and disposal rights.
+
+### Tract
+A Tract is an area of land within a title defined by locations sharing identical rights. This attribute has all of the tract(s) information contained within a title. See Tracts Appendix below for a detailed description of the information included.
```
```suggestion
A Tract is an area of land within a title defined by locations sharing identical rights. This attribute has all of the tract(s) information contained within a title. 
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld:
```diff
@@ -0,0 +1,141 @@
+    "VerifiableCredential",
+    "ExampleCredential",
+    "DigitalConformityCredential",
+    "BCTenureTitle"
```
```suggestion
    "PetroleumNaturalGasTitle"
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld:
```diff
@@ -0,0 +1,141 @@
+  "issuer": {
+    "id": "did:web:bcgov.github.io:bc-vcpedia:applications:tenure-branch",
+    "url": "https://www2.gov.bc.ca/gov/content/industry/natural-gas-oil/petroleum-natural-gas-tenure",
+    "name": "Tenure and Resource Stewardship Branch",
+    "description": "The Tenure and Resource Stewardship Branch (TRSB)"
```
```suggestion
    "name": "BC Director of Petroleum Lands",
    "description": "Director means an officer or employee of the ministry who is designated as the Director of Petroleum Lands by the minister"
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld:
```diff
@@ -0,0 +1,141 @@
+    "DigitalConformityCredential",
+    "BCTenureTitle"
+  ],
+  "issuer": {
```
```suggestion
  "issuer": {
  "type": ["Director"],
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld:
```diff
@@ -0,0 +1,141 @@
+      },
+      "scope": {
+        "type": [
+          "ConformityAssessmentScheme"
```
Discussion point - needs further clarification from UNTP regarding linkage of assessment scheme to regulation. Will assume for the purpose of demo that this artifact relates to the governance of the credential (application layer).

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld:
```diff
@@ -0,0 +1,141 @@
+        ],
+        "name": "B.C. Petroleum & Natural Gas Title - DRAFT",
+        "url": "https://bcgov.github.io/bc-vcpedia/credentials/bc-petroleum-and-natural-gas-title/governance.html",
+        "issuingBody": {
+          "name": "Tenure and Resource Stewardship Branch",
+          "description": "Energy & Mines Digital Trust (EMDT) is empowering natural resource operators with digital tools that support B.C.'s sustainable economic development while maintaining high levels of environmental protection."
+        },
```
```suggestion
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld:
```diff
@@ -0,0 +1,141 @@
+        {
+          "type": [
+            "ConformityAssessment",
+            "TenureTitle",
```
```suggestion
            "PetroleumNaturalGasTitle",
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld:
```diff
@@ -0,0 +1,141 @@
+            "url": "https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/00_96361_01",
+            "name": "Petroleum and Natural Gas Act",
+            "effectiveDate": "2024-06-02",
+            "issuingBody": {
+              "id": "https://www2.gov.bc.ca/gov/content/governments/organizational-structure/ministries-organizations/ministries/energy-mines-and-petroleum-resources",
+              "name": "The Ministry of Energy, Mines and Low Carbon Innovation"
+            }
```
```suggestion
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld:
```diff
@@ -0,0 +1,141 @@
+            ],
+            "url": "https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/00_96361_01",
+            "name": "Petroleum and Natural Gas Act",
+            "effectiveDate": "2024-06-02",
```
```suggestion
            "effectiveDate": "2024-06-02"
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+### LegalEntity
+An entity registered in orgbook.
+
+### TenureTitle
```
```suggestion
### PetroleumNaturalGasTitle
```

PatStLouis commented:
@bblazicevic @krobinsonca 
Here's the latest changes:

- renamed example files from `v0-*` to `draft-*`
- renamed `proof` to `signature` in the filenames
- changed `TenureTitle` to `BCPetroleum&NaturalGasTitle` in all relevant places (examples, vocab and context)
- re-issued the examples based on these changes

Please let me know what other changes you would like to see. 

If you want to see the examples in full, you can refer to my fork:
https://github.com/OpSecId/bc-vcpedia/blob/pstlouis/png-provide-credential-examples/credentials/bc-petroleum-and-natural-gas-title/examples/draft-no-signature.jsonld

krobinsonca commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+# Vocabulary
+
```
We should include a reference to the source of truth of these vocab entries, in this case, I believe it is the Act.

PatStLouis commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+# Vocabulary
+
```
```suggestion
_Some terms defined in this vocabulary are referencing terms defined in the [The Petroleum and Natural Gas Act (PNG Act)](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/00_96361_01#part1)._
```

PatStLouis replied:
@krobinsonca Does this work for this purpose?

PatStLouis commented on credentials/bc-petroleum-and-natural-gas-title/governance.md:
```diff
@@ -131,120 +131,39 @@ Legislation and regulations govern the disposition, administration, and manageme
-#### standardZone
-describe the different ways by which a zone can be designated for use, defining intervals as either feet or meters. Next, the log type is identified followed by the well permit number. These codes outline 1. a standard 5-digit code; 2. descriptions of this code including geographical areas and details; 3. shortened descriptions of these details; 4. effective dates; and 6. expiry dates.
+### 2.6.1.1 Issuers
+#### Tenure Branch
```
```suggestion
#### Director of Petroleum Lands
```

PatStLouis commented on credentials/bc-petroleum-and-natural-gas-title/governance.md:
```diff
@@ -131,120 +131,39 @@ Legislation and regulations govern the disposition, administration, and manageme
+[DID document](../../applications/tenure-branch/did.json)
 
+### 2.6.1.1 Examples
+[Credential](examples/v0.jsonld)
+[VC-DI](examples/v0-embedded-proof.jsonld)
+[VC-JOSE](examples/v0-envelopped-proof.jsonld)
```
```suggestion
[Credential](examples/draft-no-signature.jsonld)
[VC-DI](examples/draft-embedded-signature.jsonld)
[VC-JOSE](examples/draft-enveloped-signature.jsonld)
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+## Classes
+
+### Director
+An officer or employee of the ministry who is designated as the Director of Petroleum Lands by the minister
```
```suggestion
An officer or employee of the ministry who is designated as the Director of Petroleum Lands by the minister.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+An officer or employee of the ministry who is designated as the Director of Petroleum Lands by the minister
+
+### LegalEntity
+An entity registered in orgbook.
```
```suggestion
An entity registered in [OrgBook](https://orgbook.gov.bc.ca/search).
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+The majority of subsurface petroleum and natural gas (PNG) resources in British Columbia (B.C.) are owned by the Province. By entering into a tenure agreement with the Province, private industry can develop these resources. Tenure agreements are the mechanism used by the Province to give rights to petroleum and natural gas resources through issuance of Petroleum and Natural Gas Titles.
+
+### TitleHolder
+The PNG Act describes a "holder of a location" as meaning "in accordance with the context, a permittee, licensee" . In this credential, the term title holder is used to reference a person in whose name a PNG title document is recorded in the division records. Title holders can be registered companies, and/or persons. A Title Holder will hold a percentage (%) interest in the title. Percent interes ...
```
```suggestion
The PNG Act describes a "holder of a location" as meaning "in accordance with the context, a permittee, licensee". In this credential, the term Title Holder refers to a person whose name in a PNG title document is recorded in the division records. Title Holders can be registered companies, and/or persons. A Title Holder will hold a percent (%) interest in the title. Percent interest defines the percentage of interest allotted to each party named in the PNG title document. The percentage of ownership can be divided up to eight digits and should always equal 100% total. 
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+The PNG Act describes a "holder of a location" as meaning "in accordance with the context, a permittee, licensee" . In this credential, the term title holder is used to reference a person in whose name a PNG title document is recorded in the division records. Title holders can be registered companies, and/or persons. A Title Holder will hold a percentage (%) interest in the title. Percent interes ...
+
+### Lease
+A subsisting lease issued under this Act
```
```suggestion
A subsisting lease issued under the PNG Act.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+A subsisting lease issued under this Act
+
+### Leasee
+A person in whose name a lease is recorded in the division records
```
```suggestion
Defined by the PNG Act as "a person in whose name a lease is recorded in the division records".
```

bblazicevic replied:
```suggestion
A person in whose name a lease is recorded in the division records.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+A person in whose name a lease is recorded in the division records
+
+### Licence
+A drilling licence or a licence under section 126 (3) or 130 (3)
```
```suggestion
A drilling licence under [Part 5.1](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96361_01#part5.1) of the PNG Act.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+A drilling licence or a licence under section 126 (3) or 130 (3)
+
+### Licensee
+A person in whose name a licence is recorded in the division records
```
```suggestion
A person in whose name a licence is recorded in the division records.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+
+### NaturalGas
+All fluid hydrocarbons, before and after processing, that are not defined as petroleum, and any of the following that is produced from a well:
+carbon dioxide; helium; hydrogen; hydrogen sulphide;
```
```suggestion
carbon dioxide; helium; hydrogen; hydrogen sulphide.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+carbon dioxide; helium; hydrogen; hydrogen sulphide;
+
+### Permit
+A subsisting exploration permit issued under this Act
```
```suggestion
A subsisting exploration permit issued under the PNG Act.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+A subsisting exploration permit issued under this Act
+
+### Permitee
+The person in whose name an exploration permit is recorded in the division records
```
```suggestion
The person in whose name an exploration permit is recorded in the division records.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+The person in whose name an exploration permit is recorded in the division records
+
+### Petroleum
+Crude petroleum and all other hydrocarbons, regardless of gravity, that are or can be recovered in liquid form from a pool through a well by ordinary production methods or that are or can be recovered from oil sand or oil shale
```
```suggestion
Crude petroleum and all other hydrocarbons, regardless of gravity, that are or can be recovered in liquid form from a pool through a well by ordinary production methods or that are or can be recovered from oil sand or oil shale.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+Crude petroleum and all other hydrocarbons, regardless of gravity, that are or can be recovered in liquid form from a pool through a well by ordinary production methods or that are or can be recovered from oil sand or oil shale
+
+### PetroleumWell
+A well in which casing is run and that, in the opinion of the regulator, is producing or is capable of producing from a petroleum bearing zone
```
```suggestion
A well in which casing is run and that, in the opinion of the regulator, is producing or is capable of producing from a petroleum-bearing zone.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+A well in which casing is run and that, in the opinion of the regulator, is producing or is capable of producing from a petroleum bearing zone
+
+### Well
+A hole in the ground
+(a) made or being made by drilling, boring or any other method to obtain petroleum or natural gas,
+(b) made or being made by drilling, boring or any other method to explore for, develop or use a storage reservoir for the storage or disposal of petroleum, natural gas, carbon dioxide, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance,
+(c) used, drilled or being drilled to inject natural gas, water produced in relation to the production of petroleum or natural gas or other substances into an underground formation in connection with the production of petroleum or natural gas,
+(d) used to dispose of petroleum, natural gas, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance into a storage reservoir, or
+(e) used, drilled or being drilled to obtain geological or geophysical information respecting petroleum or natural gas,
```
```suggestion
A hole in the ground:
(a) made or being made by drilling, boring or any other method to obtain petroleum or natural gas;
(b) made or being made by drilling, boring or any other method to explore for, develop or use a storage reservoir for the storage or disposal of petroleum, natural gas, carbon dioxide, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance;
(c) used, drilled or being drilled to inject natural gas, water produced in relation to the production of petroleum or natural gas or other substances into an underground formation in connection with the production of petroleum or natural gas;
(d) used to dispose of petroleum, natural gas, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance into a storage reservoir; or
(e) used, drilled or being drilled to obtain geological or geophysical information respecting petroleum or natural gas.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+(e) used, drilled or being drilled to obtain geological or geophysical information respecting petroleum or natural gas,
+
+### Zone
+A stratum or strata designated by the regulator as a zone generally or for a designated area or a specific well, or a stratum or strata designated by the director for the purposes of the administration of petroleum and natural gas rights or storage and disposal rights.
```
```suggestion
A stratum or strata designated by the regulator as a zone generally for a designated area or a specific well, or a stratum or strata designated by the director for the purposes of the administration of petroleum and natural gas rights or storage and disposal rights.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+## Properties
+
+### interest
+an undivided interest in a location
```
```suggestion
an undivided interest in a location.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+This field specifies the type of title based on the PNG Act. Title types can include the following: 1. Lease (Part 6); 2. Permit (Part 5); 3. Drilling Licence (Part 5.1), and 4. Storage Reservoir Licence (Part 14.130). Each title has only one title type.
+
+### titleNumber
+A number is used to uniquely identify a title, assigned based on administrative policy by the TRSB. The title number is associated with the Exploration Licence outlined in Section 126, part 14.126 of the PNG Act.
```
```suggestion
A number is used to uniquely identify a title, assigned based on administrative policy by the Tenure and Resource Stewardship Branch (TRSB). The title number is associated with the Exploration Licence outlined in Section 126, Part 14.126 of the PNG Act.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+A number is used to uniquely identify an origin, assigned based on administrative policy by the TRSB.
+
+### area
+This attribute is the surface area in planned use, measured in hectares. This includes 3 dimensional layers.
```
```suggestion
This attribute is the surface area in planned use, measured in hectares. This includes three dimensional layers.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/vocabulary.md:
```diff
@@ -0,0 +1,98 @@
+This describes the length of time (term) granted for each title, expressed in YYYY. This date does not change.
+
+### caveats
+Caveats provide information and guidance to the tenure holder that will assist in activity planning by identifying potential access restrictions. Caveats will also flag concerns identified through pre-tenure consultation and may recommend engagement with First Nations, stakeholders, and other agencies as appropriate. Caveats often point to relevant statute and policy and are not binding or enforc ...
```
```suggestion
Caveats provide information and guidance to the title holder that will assist in activity planning by identifying potential access restrictions. Caveats will also flag concerns identified through pre-tenure consultation and may recommend engagement with First Nations, stakeholders, and other agencies as appropriate. Caveats often point to relevant statutes and policy and are not binding or enforced by the ministry.
```

bblazicevic commented on credentials/bc-petroleum-and-natural-gas-title/v0.jsonld:
Should we call file name draft.jsonld

PatStLouis replied:
file has been renamed to `draft.jsonld`

PatStLouis commented:
@krobinsonca @bblazicevic please bundle edits under a single review next time as we now have 15 prs each fixing a small typo. This will make the commit history difficult to read and creates a lot of noises for watchers of the repo as each commit will trigger an email.

PatStLouis commented:
When you make an edit, you have the option to `submit the comment` or `start a review`. By clicking start a review, it will group all edits you are making, and then when you are satisfied you can click the button submit review at the top right corner of the page. This will group all edits/comments as part of 1 review instead of individual requests.

PatStLouis commented on credentials/bc-petroleum-and-natural-gas-title/governance.md:
```diff
@@ -131,120 +131,39 @@ Legislation and regulations govern the disposition, administration, and manageme
 
 ### 2.6.1.1 Vocabulary
+[Vocabulary file](vocabulary.md)
+[Context file](v0.jsonld)
```
```suggestion
[Context file](context.jsonld)
```

krobinsonca reviewed (approved):
Approved
